<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>641</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_x0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>x0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_x1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>x1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Çöz</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_durum">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Değerleri Giriniz.</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
